Dockerfile

```diff
@@ -0,0 +1,14 @@
+# USAGE EXAMPLES
+# docker build -t insider .
+# docker run -ti --rm insider -tech android -target $pwd
+# docker run -ti --rm insider -tech javascript -target <myprojectfolder>
+# docker run -ti --rm insider -tech=android -target=<myandroidfolder>
+# docker run -ti --rm insider -tech android -target <myfolder> -no-html
+
+FROM  alpine
+
+    ENV VERSION 2.0.5
```

Comment:
Can you remove this indentation? I guess that Dockerfile has no indention like this. If yes, we can merge, seems fine to me.